DependentExtensions/Lobby2/Lobby2Message.h

```diff
@@ -321,7 +321,7 @@ struct Lobby2Message
 	/// Print the result of DebugMsg
 	virtual void DebugPrintf(void) const
 	{
-		SLNet::RakString out; DebugMsg(out); printf(out.C_String());
+		SLNet::RakString out; DebugMsg(out); puts(out.C_String());

```

Comment:
Does `printf` really inflicts compilation errors? Maybe adding something like missing `#include <cstdio>` is a better solution?

Reply:
It only because of the security warning caused by a pointer to arbitrary string used as format instead of a fixed string literal.